src/lightning_app/runners/cloud.py

```diff
@@ -329,6 +332,20 @@ def dispatch(
                         default=True,
                     )
                     app_config.cluster_id = None
+                if existing_instance and existing_instance.spec.cluster_id != app_config.cluster_id:
+                    raise ValueError(
+                        dedent(
+                            f"""\
+                            Can not run app '{app_config.name}' on cluster {app_config.cluster_id} since it already exists on {existing_instance.spec.cluster_id}
+                                (moving apps between clusters is not supported).
```

Comment:
some terms are in quotes, others are not

Reply:
Making this consistent. Thanks!